build/cli/src/lib.rs

```diff
@@ -834,7 +836,18 @@ pub async fn main_internal(config: Option<enso_build::config::Config>) -> Result
                 .run_ok()
                 .await?;
 
+            // Copied from `build` in `lib/rust/ensogl/pack/src/lib.rs`.
```

Comment:
1. What is copied from `lib/rust/ensogl/pack/src/lib.rs`? The next line? Several next lines? I don't understand this comment. Anyway, if several next lines are the same as logic in `lib/rust/ensogl/pack/src/lib.rs` then this Is bad, as replicating the same logic in multiple places makes the code very hard to maintain.
2. There is no explanation what this code does. It looks complex.